### ISSUE TITLE
Reduce braveryPanelBodyList height to avoid the vertical scroll bar from appearing

### ISSUE THREAD

**Issue by luixxiul** [brave/browser-laptop#12041]
### Description

Reduce braveryPanelBodyList height to avoid appearing scroll bar.

This is a follow-up to https://github.com/brave/browser-laptop/pull/11880#issuecomment-345629718

### Steps to Reproduce

  1. Open about about:preferences#shields
  2. Disable the compact panel option if it is enabled
  3. Open https://www.youtube.com/watch?v=UW0-eHRSyzQ
  4. Click the lion icon
  5. Click the blocked ads count

**Actual result:**

Sometimes the vertical scroll bar appears on a case where it has not until 0.19.x.

**Expected result:**

The scroll bar should not appear on that case.

Reducing the same height as the new column where fingerprinting protection options are displayed, the scroll bar will not appear.

**Reproduces how often:**

### Brave Version

**about:brave info:**

Since `0.20.x`

**Reproducible on current live release:**

No.

**Comment by luixxiul** [comment]
This needs investigation after merging.

**Comment by srirambv** [comment]
Still an issue on 0.20.9. Entire shields panel shift across by 1px due to the vertical scrollbar
![shields-horizontal bar](https://user-images.githubusercontent.com/17010094/34266699-16244624-e672-11e7-95f3-113203425889.gif)

**Comment by luixxiul** [comment]
Closing as wontfix. We probably should have a better fix to avoid the scroll bars than just changing the height with magic number.
